corelib/src/starknet/syscalls.cairo

```diff
@@ -8,23 +8,23 @@ use starknet::contract_address::ContractAddress;
 // `entry_point_selector` - A selector for a function within that contract.
 // `calldata` - Call arguments.
 extern fn call_contract_syscall(
-    address: ContractAddress, entry_point_selector: felt252, calldata: Array<felt252>
-) -> SyscallResult<Array<felt252>> implicits(GasBuiltin, System) nopanic;
+    address: ContractAddress, entry_point_selector: felt252, calldata: Span<felt252>
+) -> SyscallResult<Span<felt252>> implicits(GasBuiltin, System) nopanic;
```

Comment:
I think the return type should be Array. The called contract has no use for it, so it might as well give its ownership